definitions/tools/survivor.cwl

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Run SURVIVOR to merge SV calls"
+
+baseCommand: ["/bin/bash", "/usr/bin/survivor_merge_helper.sh"]
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/survivor-cwl:1.0.6.1"
+    - class: ResourceRequirement
+      ramMin: 2000
+      coresMin: 1
+
+inputs:
+    vcfs:
+        type: File[]
+        inputBinding:
+            position: 1
+            itemSeparator: ","
+        doc: "Array of VCFs to merge SV calls of"
+    max_distance_to_merge:
+        type: int
+        inputBinding:
+            position: 2
+        doc: "Maximum distance of variants to consider for merging"
+    minimum_sv_calls:
+        type: int
+        inputBinding:
+            position: 3
+        doc: "Minimum number of sv calls needed to be merged"
+    same_type:
+        type: int
+        inputBinding:
+            position: 4
+        doc: "Require merged SVs to be of the same type, 1=yes"
```

Comment:
Is anything else `no`?

Reply:
it's 1=yes, 0=no. updated documentation to state that. I was going to use the boolean input type but thought that was for flag inputs. this has to have something as all inputs are positional

Reply:
Yeah, the boolean type is mostly for flag inputs.  You could use JavaScript to convert it to a `1` or a `0`, or possibly define an enumerated type here that accepts only `0` and `1`.